setup.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+setup(name='python-bitflow',
+      version='0.1',
+      description='bitflow is a stream processing framework that can be controlled through a script language.',
```

Comment:
Maybe fix the description, that python-bitflow is intended to be used as external executable in go-bitflow? And add real email? Why is the 'python-bitflow' entry in scripts doubled?